shell/platform/android/android_context_gl.h

```diff
@@ -43,13 +45,26 @@ class AndroidEGLSurface {
   ///
   bool MakeCurrent() const;
 
+  //----------------------------------------------------------------------------
+  /// @brief      This is the minimal area that needs to be repainted to get
```

Comment:
This doc can be elaborated to explain that this buffer's age might be behind the other buffers in flight and that this rect accounts for the accumulated damage.